### PR TITLE
Bug 1371446 - Don't allow needinfo requests from the void

### PR DESCRIPTION
**Pull request by umohm1**

### CODE REVIEW

File: extensions/Needinfo/template/en/default/bug/needinfo.html.tmpl [hunk 1]
```diff
@@ -91,12 +91,14 @@ $(function() {
     if (role == 'other') {
       $('#needinfo_from_container').removeClass('bz_default_hidden');
       $('#needinfo_from').prop('disabled', false);
-      $('#needinfo_role_identity').text('');
+      $('#needinfo_role_identity').text
```
Comment (floatingatoll) [inline extensions/Needinfo/template/en/default/bug/needinfo.html.tmpl]: How does this single specific change interact with the two 'required' property changes below, on either side of the if/else?

Comment (dylanwh) [inline extensions/Needinfo/template/en/default/bug/needinfo.html.tmpl]: I'm pretty sure this is a bug.

File: extensions/Needinfo/template/en/default/bug/needinfo.html.tmpl [hunk 2]
```diff
@@ -91,12 +91,14 @@ $(function() {
...
+      $('needinfo_from_textfield').prop('required', true);
```
Comment (dylanwh) [inline extensions/Needinfo/template/en/default/bug/needinfo.html.tmpl]: also $('#needinfo_role_identity').text('')

File: extensions/Needinfo/template/en/default/bug/needinfo.html.tmpl [hunk 3]
```diff
@@ -92,11 +92,13 @@ $(function() {
       $('#needinfo_from_container').removeClass('bz_default_hidden');
       $('#needinfo_from').prop('disabled', false);
       $('#needinfo_role_identity').text('');
+      $('needinfo_from').prop('required', true);
```
Comment (dylanwh) [inline extensions/Needinfo/template/en/default/bug/needinfo.html.tmpl]: this selector doesn't match anything -- we'll have a chat to figure this out.

**Comment by dylanwh** [comment]
I have some changes in https://github.com/umohm1/bmo/pull/2 that fix the red border thing we talked about on irc -- merge that in and I think this is done.

**Pull request merged**